pkg/kv/kvclient/rangefeed/config.go

```diff
@@ -287,3 +293,28 @@ func WithPProfLabel(key, value string) Option {
 		c.extraPProfLabels = append(c.extraPProfLabels, key, value)
 	})
 }
+
+// WithUseMuxRangeFeed configures rangefeed to use MuxRangeFeed RPC.
+func WithUseMuxRangeFeed() Option {
+	return optionFunc(func(c *config) {
+		c.rfOpts = append(c.rfOpts, kvcoord.WithMuxRangeFeed())
+	})
+}
+
+// metamorphicOptions initializes set of Options with metamorphic values.  These
+// options are applied to the config before user specified options are applied.
+var metamorphicOpts = func() (opts []Option) {
+	if util.ConstantWithMetamorphicTestBool("use-mux", true) {
+		opts = append(opts, WithUseMuxRangeFeed())
+	}
+	opts = append(opts, WithInitialScanParallelismFn(func() int {
+		return util.ConstantWithMetamorphicTestRange("scan-parallelism",
+			1, 1, runtime.GOMAXPROCS(0))
+	}))
+	retryBehavior := util.ConstantWithMetamorphicTestChoice(
+		"scan-retry-behavior", ScanRetryAll, ScanRetryRemaining).(ScanRetryBehavior)
```

Comment:
Can this change the observed behavior and make tests flaky?

Reply:
I'm still working through the flakes in this PR; so.. perhaps.
The intention, of course was for this to not cause flakes.
If this particular option causes flakes, then the test can be explicit and specify retry behavior.
Otherwise, they are saying: we don't care, just as long as you return all the values, at least once.